### DISCUSSION TITLE
Cannot convert a BigInt value to a number

### DISCUSSION THREAD

**Discussion by RoneyCBR** [Web3Auth/Web3Auth#1203] [Help]
Hello, can you help me with a problem that I have, I have a project with web3auth. locally it runs fine, but when building the application and mounting it on a server it gives me the following error:
"Cannot convert a BigInt value to a number."

This does not happen to me when running the application in my local environment.

any idea how to solve it?

I attached a screenshot

![image](https://user-images.githubusercontent.com/84111606/212386375-33cf847c-7e02-4d01-8439-a7150fce4250.png)

Thank you.

**Comment by Saviour1001** [comment]
Hey there, we have documented this issue on our troubleshooting page. This error can be solved by adding specified versions of the browsers in `package.json`. 
Add this in your `package.json`
```
"browserslist": {
    "production": [
      "chrome >= 67",
      "edge >= 79",
      "firefox >= 68",
      "opera >= 54",
      "safari >= 14"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  }
```
You can read more about it [here](https://web3auth.io/docs/troubleshooting/react-big-int-error).
